test/blackbox/common/RTPSBlackboxTestsPersistence.cpp

```diff
@@ -145,7 +145,7 @@ class Persistence : public ::testing::TestWithParam<communication_type>
         const int32_t pid = GET_PID();
         memcpy(guid_prefix_.value + 4, &pid, sizeof(pid));
         guid_prefix_.value[8] = HAVE_SECURITY;
-        guid_prefix_.value[9] = 3; //PREALLOCATED_MEMORY_MODE
```

Comment:
Is this change motivated by a test failure? `3` does not correspond to the numeric value of `PREALLOCATED_MEMORY_MODE` in the `MemoryManagementPolicy` enumerator. Consequently, I am for removing the comment and leaving the `3` (the guidPrefix can be set by the user directly).